.lando.yml

```diff
@@ -116,7 +116,7 @@ tooling:
       - appserver: echo "Clearing cache..."
       - appserver: drush cr -y
       - node: echo "Building theme asset files..."
-      - node: cd /app/web/themes/custom/cloudy && yarn run build
+      - node: cd /app/web/themes/custom/cloudy && yarn run build --frozen-lockfile
```

Comment:
The build command doesn't do any installs, so this flag doesn't do anything here. I will remove it, but we do want to add it to the start command. I'm going to replace the start command in the other PR as well.